src/experiment_prototype/experiment_slice.py

```diff
@@ -506,6 +505,16 @@ def check_cfs_scheme(cls, cfs_scheme, values):
                             f"adjust the cfs_range values of the experiment."
                         )
 
+            for stage, dm_rate in enumerate(cfs_scheme.dm_rates):
+                if (
+                    cfs_scheme.input_rates[stage]
+                    > dm_rate * cfs_scheme.output_rates[stage]
+                ):
+                    raise ValueError(
+                        f"CFS scheme decimation stage {stage} is aliasing. Ensure that input_rate/output_rate "
+                        f"is greater than the decimation rate."
+                    )
+
```

Comment:
I think this check would be better suited to the DecimationStage object in decimation_scheme.py